### DISCUSSION TITLE
Prediction Error Plot using any other evaluation metric than R2 in regression?

### DISCUSSION THREAD

**Discussion by dryogibansal** [pycaret/pycaret#1691] [Q&A] [answered]
In regression, while using the plot_model function, the error plot shows the prediction errors using R2 evaluation metric. Is there a possibility to plot the prediction errors using another evaluation metric let's say MAE or RMSE?

**Comment by moezali1** [comment, marked as answer]
@vickaryan If you are asking about this one:

![image](https://user-images.githubusercontent.com/54699234/137587353-d1dd2b34-8225-4985-b8ab-a1f14c5bac25.png)

It's not possible. PyCaret is using this implementation: https://www.scikit-yb.org/en/latest/api/regressor/peplot.html

**Reply by dryogibansal** [reply]
ok. Thanks for a quick response.
